repos/dir.targets

```diff
@@ -322,6 +322,34 @@
                                OverwriteDestination="true" />
   </Target>
 
+  <Target Name="EnsurePackagesCreated"
+          AfterTargets="CopyPackage"
+          Condition="'$(SkipEnsurePackagesCreated)' != 'true'">
+    <ItemGroup>
+      <JustSourceBuiltPackages
+        Include="$(SourceBuiltPackagesPath)*.nupkg"
+        Exclude="
+          $(SourceBuiltPackagesPath)*.symbols.nupkg;
+          @(PreviouslySourceBuiltPackages)" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <_PackagesNotCreatedReason Include="^ Silent submodule build failure: check $(RepoConsoleLogFile)" />
```

Comment:
I think this could be a bit more clear that these are just possible causes, maybe something like
- "This could be a silent submodule build failure..."
- "If $(RepositoryName) doesn't build nuget packages..."

Reply:
True, I was originally thinking they'd show up more clearly as one error message, but since they're split up it's a bit confusing. Thanks.